example-project/src/SceneSwitcher.ts

```diff
@@ -27,25 +20,32 @@ export class SceneSwitcher extends ReninNode {
 
     this.children = children<{
       spinningcube: SpinningDonut;
+      flatland: FlatLand;
       jumpingbox: JumpingBox;
     }>({
       spinningcube: new SpinningDonut(renin),
+      flatland: new FlatLand(renin),
       jumpingbox: new JumpingBox(renin),
     });
     this.scene.add(this.screen);
     this.scene.add(this.camera);
     this.camera.position.z = 10;
   }
 
-  public render(frame: number, renderer: WebGLRenderer, renin: Renin) {
+  public render(frame: number) {
     this.screen.material.map = null;
     if (this.children?.jumpingbox.isActive) {
       //@ts-ignore
       this.screen.material.map = this.children.jumpingbox.renderTarget.texture;
     }
-    if (this.children?.spinningcube.isActive) {
+    if (this.children?.flatland.isActive) {
       //@ts-ignore
-      this.screen.material.map = this.children.spinningcube.renderTarget.texture;
+      this.screen.material.map = this.children.flatland.texture;
+    }
+    if (this.children?.spinningcube.isActive) {
+      this.screen.material.map =
+        //@ts-ignore
+        this.children.spinningcube.renderTarget.texture;
     }
     this.screen.material.needsUpdate = true;
     renderer.setRenderTarget(this.renderTarget);
```

Comment:
Crashing on `renderer` not being defined.

Reply:
oops, accidental removal of renderer method argument